internal/cmd/flink/command_statement.go

```diff
@@ -0,0 +1,86 @@
+package flink
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+type statementOut struct {
+	Name         string `human:"Name" serialized:"name"`
+	Statement    string `human:"Statement" serialized:"statement"`
+	ComputePool  string `human:"Compute Pool" serialized:"compute_pool"`
+	Status       string `human:"Status" serialized:"status"`
+	StatusDetail string `human:"Status Detail,omitempty" serialized:"status_detail,omitempty"`
+}
+
+func (c *command) newStatementCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "statement",
+		Short: "Manage Flink SQL statements.",
+	}
+
+	cmd.AddCommand(c.newStatementDeleteCommand())
+	cmd.AddCommand(c.newStatementListCommand())
+
+	return cmd
+}
+
+func (c *command) addComputePoolFlag(cmd *cobra.Command) {
+	cmd.Flags().String("compute-pool", "", "Flink compute pool ID.")
+
+	pcmd.RegisterFlagCompletionFunc(cmd, "compute-pool", func(cmd *cobra.Command, args []string) []string {
```

Comment:
```suggestion
	pcmd.RegisterFlagCompletionFunc(cmd, "compute-pool", c.autocompleteComputePools)
```
The func here looks to be a duplicate of the code in `c.validComputePoolArgs`. Suggestion: put the code in a new `func (c *command) autocompleteComputePools` to reduce code duplication.